pyzm/ml/detect_sequence.py

```diff
@@ -53,9 +53,23 @@ def __init__(self, options={}, global_config={}):
                             },
 
                             # within object, this is a sequence of object detection libraries. In this case,
-                            # I want to first try on my TPU and if it fails, try GPU
-                            'sequence': [{
-                                #First run on TPU
+                            # I want to first try on AWS Rekognition, then on Coral TPU and if it fails, try GPU
```

Comment:
Let's move this example blurb to after the local instances - the reason here is this is a paid service. Many ZM users will not want to run this first if cost is a concern. A lot of users are generating hundred of events per hour with several frame sets, so costs will easily go up to a few hundred dollars (at least) a month

Reply:
It still has to be configured with credentials first, but sure I'll move it down.

Reply:
Done.